optaplanner-docs/src/main/asciidoc/PlannerConfiguration/PlannerConfiguration-chapter.adoc

```diff
@@ -1738,6 +1738,25 @@ For example in Logback, use a `SiftingAppender` in ``logback.xml``:
 ----
 ====
 
+[[monitoring]]
+=== Monitoring
+
+OptaPlanner exposes metrics via https://micrometer.io/[Micrometer] which can be used for monitoring the Solver. OptaPlanner automatically connect to configured registries when used in Quarkus or Spring Boot. In plain Java, add the registry to the global Metrics object:
+[source,java,nowrap]
+----
+Metrics.addRegistry(new SimpleMeterRegistry());
+----
+
+The following metrics are exposed (format may differ depending on registry):
+
+- `optaplanner.solver.errors`: Count the total number of errors that occurred while solving
+
+- `optaplanner.solver.solve-length.active-count`: The number of solvers currently active
+
+- `optaplanner.solver.solve-length.seconds-max`: How long the oldest active solver been running
+
+- `optaplanner.solver.solve-length.seconds-duration-sum`: The sum of each active solver solve length (ex: if there are two active solvers, one been running for 3 minutes, the other for 1 minute, this would be 4 minutes).
```

Comment:
```suggestion
- `optaplanner.solver.solve-length.seconds-duration-sum`: The sum of each active solver solve length (ex: if there are two active solvers, one running for 3 minutes and the other for 1 minute, this would be 4 minutes).
```